test/src/integration.long/transactions.test.ts

```diff
@@ -83,9 +86,35 @@ describe("transactions", function() {
             }
         });
 
-        // Not implemented
-        it("mint amount U64 max");
-        it("mint amount exceeds U64");
+        it("Mint unsuccessful - mint amount U64.MAX_VALUE + 1", async function() {
+            const scheme = node.sdk.core.createAssetScheme({
+                shardId: 0,
+                metadata: "",
+                amount: 0
+            });
+            (scheme.amount.value as any) = U64.MAX_VALUE.value.plus(1);
```

Comment:
What's the value of `U64.MAX_VALUE.value.plus(1)`?

Reply:
`U64` internally uses `BigNumber`, which does not have any limitation. This is the code to send `U64.MAX_VALUE + 1` by a trick.